de.js

```diff
@@ -0,0 +1,334 @@
+    edit_profile: `Profil bearbeiten`,
+    social_connect: `Soziale Verbindung`,
+    profile_privacy: `Profil Datenschutz`,
+    profile_is_public: `Profil ist öffentlich`,
+    show_wallet_address_and_tip_button: `Wallet-Adresse und Trinkgeld-Button anzeigen`,
+    preferences: `Einstellungen`,
+    confirm_x_tips: `Bestätigen Sie 𝕏 Tipps`,
+    tips_received: `Erhaltene Trinkgelder`,
+    new_followers: `Neue Follower`,
+    new_memes: `Neue Memes`,
+    about_mydoge: `Über MyDoge`,
+    terms_of_use: `Nutzungsbedingungen`,
+    privacy_policy: `Datenschutzrichtlinie`,
+    share_mydoge: `MyDoge teilen`,
+    visit_our_website: `Besuche unsere Webseite`,
+    settings_and_support: `Einstellungen und Support`,
+    connect_twitch: `Twitch verbinden`,
+    connect_x: `Verbinden 𝕏`,
+    create_a_post: `Einen Beitrag erstellen`,
+    following: `Gefolgt`,
+    followers: `Anhänger`,
+    recent_activity: `Letzte Aktivität`,
+    history: `Geschichte`,
+    price: `Preis`,
+    usd: `US Dollar`,
+    txs: `TXs`,
+    hr: `Std`,
+    send: `Schicken`,
+    receive: `Erhalten`,
+    buy: `Kaufen`,
+    scan: `Scan`,
+    from: `Aus`,
+    to: `Zu`,
+    note: `Notiz`,
+    timestamp: `Zeitstempel`,
+    confirmations: `Bestätigungen`,
+    tx_hash: `TX-Hash`,
+    tx_hash_copied: `Transaktions-Hash kopiert`,
+    note_copied: `Notiz kopiert`,
+    address_copied: `Adresse kopiert`,
+    wallet: `Geldbörse`,
+    dogeverse: `Dogeverse`,
+    shop: `Geschäft`,
+    receive_doge: `Doge erhalten`,
+    send_doge: `Senden Sie Doge`,
+    confirm_transaction: `Transaktion bestätigen`,
+    scan_qr_code: `QR-Code scannen`,
+    new_post: `Neuer Beitrag`,
+    send_doge_alert_box_message: `Sie können DOGE an Ihre Freunde senden, eine Wallet-Adresse oder einen MyDoge-Benutzernamen eingeben`,
+    wallet_address_or_username: `Wallet-Adresse oder Benutzername`,
+    saved_addresses: `Gespeicherte Adressen`,
+    add_new: `Neue hinzufügen`,
+    sending_to: `Senden an`,
+    save: `Speichern`,
+    saved: `Gerettet`,
+    balance: `Gleichgewicht`,
+    max: `Max`,
+    next: `Nächste`,
+    network_fee: `Netzwerkgebühr`,
+    max_tx_size_warning: `ACHTUNG: Maximale Transaktionsgröße erreicht. Das ist das Maximum, das Sie auf einmal senden können.`,
+    confirm_transaction_warning: `Überprüfen Sie vor der Bestätigung immer doppelt, an wen Sie senden. Senden Sie niemals an eine Auszahlungsadresse einer Börse. Dogecoin-Transaktionen können nicht rückgängig gemacht werden!`,
+    add_note: `Notiz hinzufügen`,
+    wallet_address: `Wallet-Adresse`,
+    wallet_label: `Brieftaschenetikett`,
+    save_changes: `Änderungen speichern`,
+    save_address: `Adresse speichern`,
+    enter_new_PIN: `Neue PIN eingeben`,
+    confirm_new_pin: `Neue PIN bestätigen`,
+    enter_new_PIN_description: `Geben Sie Ihre neue PIN für Ihr MyDoge-Wallet ein`,
+    enter_old_PIN: `Alte PIN eingeben`,
+    enter_old_PIN_description: `Geben Sie Ihre alte PIN für Ihr MyDoge-Wallet ein`,
```

Comment:
`    enter_old_PIN_description: 'Geben Sie Ihre alte PIN für Ihre MyDoge-Wallet ein',`

Wallet in German commonly used as a female noun